### DISCUSSION TITLE
[WASM AOT] 404 error on uno-assets.txt?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10603] [Q&A] [answered]
I'm ripping through browser console errors and see this one:
`VM148:1          GET https://stage.unusual-suite.com/package_39e2eda6c76d240c639ef9b04d315d042df3080a/uno-assets.txt 404`

This is latest UNO dev bits

How could that be resolved?

**Comment by jeromelaban** [comment]
Thanks for the report. You'll need to include `.txt` as a known mime type for your server.

**Reply by DierkDroth** [reply]
@jeromelaban this is weird. There is something not right with building and deploying the AOT packages.

I get 404 errors e.g. on these links:
https://stage.unusual-suite.com/app/wasm/Latest/Assets/Unusual.png
https://stage.unusual-suite.com/app/wasm/package_1bc4b6d9a64b1214aad5ef4e9fc08c9e933cba4f/uno-assets.txt
https://stage.unusual-suite.com/app/wasm/package_1bc4b6d9a64b1214aad5ef4e9fc08c9e933cba4f/Assets/UnusualSuite.png

However, these files just don't exist at that location. Their real locations are:
https://stage.unusual-suite.com/app/wasm/Latest/package_1bc4b6d9a64b1214aad5ef4e9fc08c9e933cba4f/Assets/Unusual.png
https://stage.unusual-suite.com/app/wasm/Latest/package_1bc4b6d9a64b1214aad5ef4e9fc08c9e933cba4f/uno-assets.txt
https://stage.unusual-suite.com/app/wasm/Latest/package_1bc4b6d9a64b1214aad5ef4e9fc08c9e933cba4f/Assets/UnusualSuite.png
(note: don't try to click on the links. I already uploaded a new build with different package folder names in order to chase the other issues...)

In case this is not clear: I'm not manipulating the files nor folders as they are created by the build process. I'm just uploading the complete folder.

Any idea what might be going wrong?

**Reply by jeromelaban** [reply]
Nothing obvious comes to mind. If the files are present on disk, and if you try to get their content and you get a 404, this is a server configuration issue.

**Reply by DierkDroth** [reply]
@jeromelaban not sure if we're on the same page: the links in the first group **are used** by UNO WASM (and fail), the links in the 2nd group are the correct links - and work as you clicked on them as the build was uploaded 

This has nothing to do with mimetypes, since they're working on principle (see links in 2nd group)

Here are the links from **the current build** (which I'll overwrite when the next build process is done, to resume my work on the other issues).
I get 404 errors e.g. on these links:
https://stage.unusual-suite.com/app/wasm/Latest/Assets/Unusual.png
https://stage.unusual-suite.com/app/wasm/package_beacc1c0793b1dbd993fade0fbc12f8d2f1d8fa4/uno-assets.txt
https://stage.unusual-suite.com/app/wasm/package_beacc1c0793b1dbd993fade0fbc12f8d2f1d8fa4/Assets/UnusualSuite.png

However, these files just don't exist at that location. Their real locations are:
https://stage.unusual-suite.com/app/wasm/Latest/package_beacc1c0793b1dbd993fade0fbc12f8d2f1d8fa4/Assets/Unusual.png
https://stage.unusual-suite.com/app/wasm/Latest/package_beacc1c0793b1dbd993fade0fbc12f8d2f1d8fa4/uno-assets.txt
https://stage.unusual-suite.com/app/wasm/Latest/package_beacc1c0793b1dbd993fade0fbc12f8d2f1d8fa4/Assets/UnusualSuite.png

**Edit:**
@jeromelaban I had to upload a new build. I saved the old build and moved /Latest/ -> /Latest.old/
So you still could check on the PNG and TXT mimetype by editing the links accordingly.

**Comment by DierkDroth** [comment, marked as answer]
There are several issues intertwined. Closing the case in favor of [the one](https://github.com/unoplatform/uno/discussions/10616) which I was able to isolate.
